Lib/http/cookies.py

```diff
@@ -447,7 +447,7 @@ def OutputString(self, attrs=None):
     )                                # End of group 'val'
     )?                             # End of optional value group
     \s*                            # Any number of spaces.
-    (\s+|;|$)                      # Ending either at space, semicolon, or EOS.
+    (\s+|;|,|$)                    # Ending either at space, comma, semicolon, or EOS.
```

Comment:
Please add a note why the comma is added.
It is a not-RFC extension, I think we have to clarify behavior.
Issue number would be good also.

Reply:
Thanks, I added a comment.